data/core/terrain-graphics/new-macros.cfg

```diff
@@ -47,13 +47,13 @@ base
     # Places {IMAGESTEM}[1-11].png as overlay tiles.
 
 #arg LAYER
-0#endarg
+    0#endarg
```

Comment:
I don't think this should be indented. Perhaps we need to add some new rules to wmllindent?

Reply:
I think @gfgtdf got it when doing b21c110